rlclientlib/factory_resolver.cc

```diff
@@ -56,12 +56,14 @@ namespace reinforcement_learning {
   int vw_model_create(m::i_model** retval, const u::configuration&, i_trace* trace_logger, api_status* status);
   int null_tracer_create(i_trace** retval, const u::configuration&, i_trace* trace_logger, api_status* status);
   int console_tracer_create(i_trace** retval, const u::configuration&, i_trace* trace_logger, api_status* status);
+  int binding_tracer_create(i_trace** retval, const u::configuration& cfg, i_trace* trace_logger, api_status* status);
```

Comment:
> cfg [](start = 69, length = 4)

please conform with other definitions.

Reply:
Fixed.

---
In reply to: [252721167](https://github.com/VowpalWabbit/reinforcement_learning/pull/38#discussion_r252721167) [](ancestors = 252721167)